secretmanager/src/test/java/secretmanager/regionalsamples/SnippetsIT.java

```diff
@@ -0,0 +1,479 @@
+
+    assertEquals("my super secret data", secretPayload.getData().toStringUtf8());
+  }
+
+  @Test
+  public void testAddRegionalSecretVersion() throws IOException {
+    SecretName name = SecretName.parse(TEST_REGIONAL_SECRET_WITH_VERSIONS.getName());
+    SecretVersion secretVersion = AddRegionalSecretVersion.addRegionalSecretVersion(
+        name.getProject(), name.getLocation(), name.getSecret());
+    SecretVersionName secretVersionName = SecretVersionName.parse(secretVersion.getName());
+    
+    assertEquals(TEST_REGIONAL_SECRET_WITH_VERSIONS.getName(), 
+        SecretName.ofProjectLocationSecretName(
+          secretVersionName.getProject(), 
+          secretVersionName.getLocation(), 
+          secretVersionName.getSecret()).toString());
+  }
+
+  @Test
+  public void testCreateRegionalSecret() throws IOException {
+    SecretName name = TEST_REGIONAL_SECRET_TO_CREATE_NAME;
+    Secret secret = CreateRegionalSecret.createRegionalSecret(
+        name.getProject(), name.getLocation(), name.getSecret());
+    SecretName createdSecretName = SecretName.parse(secret.getName());
+    assertEquals(name.getSecret(), createdSecretName.getSecret());
+  }
+
+  @Test
+  public void testDeleteRegionalSecret() throws IOException {
+    SecretName name = SecretName.parse(TEST_REGIONAL_SECRET_TO_DELETE.getName());
+    DeleteRegionalSecret.deleteRegionalSecret(
+        name.getProject(), name.getLocation(), name.getSecret());
+
+    DeleteSecretRequest request = 
+        DeleteSecretRequest.newBuilder()
+        .setName(TEST_REGIONAL_SECRET_TO_DELETE.getName()).build();
+
+    SecretManagerServiceSettings secretManagerServiceSettings =
+        SecretManagerServiceSettings.newBuilder().setEndpoint(REGIONAL_ENDPOINT).build();
+    try (SecretManagerServiceClient client = 
+        SecretManagerServiceClient.create(secretManagerServiceSettings)) {
+      assertThrows(
+          com.google.api.gax.rpc.NotFoundException.class,
+          () -> client.deleteSecret(request));
+    }
+  }
+
+  @Test
+  public void testDeleteRegionalSecretWithEtag() throws IOException {
+    SecretName name = SecretName.parse(TEST_REGIONAL_SECRET_TO_DELETE_WITH_ETAG.getName());
+    String etag = TEST_REGIONAL_SECRET_TO_DELETE_WITH_ETAG.getEtag();
+    DeleteRegionalSecretWithEtag.deleteRegionalSecretWithEtag(
+        name.getProject(), name.getLocation(), name.getSecret(), etag);
+        
+    DeleteSecretRequest request = 
+        DeleteSecretRequest.newBuilder()
+        .setName(TEST_REGIONAL_SECRET_TO_DELETE_WITH_ETAG.getName()).build();
+
+    SecretManagerServiceSettings secretManagerServiceSettings =
+        SecretManagerServiceSettings.newBuilder().setEndpoint(REGIONAL_ENDPOINT).build();
+    try (SecretManagerServiceClient client = 
+        SecretManagerServiceClient.create(secretManagerServiceSettings)) {
+      assertThrows(
+          com.google.api.gax.rpc.NotFoundException.class,
```

Comment:
You could do an import of this class and just reference `NotFoundException.class`

Reply:
Fixed